pkgs/os-specific/linux/nvidia-x11/settings.nix

```diff
@@ -25,7 +25,7 @@ let
       cd src/libXNVCtrl
     '';
 
-    makeFlags = nvidia_x11.makeFlags ++ [
+    makeFlags = nvidia_x11.kernel.makeFlags ++ [
```

Comment:
Are these kernel.makeFlags even required, I thought they are only useful for kernel modules.

Reply:
The original change that introduced these says these need to match the compiler the kernel is built with: https://github.com/NixOS/nixpkgs/pull/136247